xbmc/pvr/channels/PVRChannelGroup.cpp

```diff
@@ -337,13 +338,21 @@ CPVRChannelNumber CPVRChannelGroup::GetChannelNumber(const CPVRChannelPtr &chann
   return member.channelNumber;
 }
 
+CPVRChannelNumber CPVRChannelGroup::GetClientChannelNumber(const CPVRChannelPtr& channel) const
```

Comment:
For new/edited code lines, I prefer using the real type (here: `std::shared_ptr<CPVRChannel>`) over typedef from `PVRTypes.h` (here: `CPVRChannelPtr`). I want to get rid of `PVRTypes.h`

Reply:
I like it. Makes sense. Done